### ISSUE TITLE
Developers get a confusing warning about Sphinx docs

### ISSUE THREAD

**Issue by migetman9** [magfest/sideboard#47]
I ssh'd onto my Ubuntu server. Cloned this repo, ran `paver make_venv` followed by `paver install_deps` and when I use `./env/bin/python sideboard/run_server.py` I get a code-hang with the text: 
`CherryPy Checker:
u'/root/sideboard/sideboard/docs/html' (root + dir) is not an existing filesystem path.
section: [/docs]
root: None
dir: u'/root/sideboard/sideboard/docs/html'`

**Comment by EliAndrewC** [comment]
Two points of clarification:

1) This isn't a code hang; you're running the server in the foreground, so that process will block indefinitely by design.

2) That error is merely a warning letting you know that you haven't built the docs via ``make html`` in the ``sphinx`` directory.

I'll update this issue description to be more precise about what is happening.

**Comment by migetman9** [comment]
Thank you for informing me! I do not know why I hadn't considered that it was running. T__T

**Comment by kitsuta** [comment]
I don't think the issue needs to be closed -- if I saw this error I would totally think something had gone horribly wrong. The message should be more clear on what's happened.

**Comment by EliAndrewC** [comment]
Agreed - that's why I renamed the title for clarity instead of closing it outright.
